src/mapkit.cs

```diff
@@ -327,6 +331,17 @@ interface MKMapItem : NSSecureCoding
 		[Export ("openMapsWithItems:launchOptions:"), Internal]
 		bool _OpenMaps ([NullAllowed] MKMapItem [] mapItems, [NullAllowed] NSDictionary launchOptions);
 
+		[iOS (13, 2), NoMac, NoTV, NoWatch]
+		[Introduced (PlatformName.UIKitForMac, 13, 2)]
+		[Export ("openInMapsWithLaunchOptions:fromScene:completionHandler:"), Internal]
```

Comment:
This API is internal, and you're not exposing it through any manually written API, so it's not usable for customers. Did you forget to add anything to this file: https://github.com/xamarin/xamarin-macios/blob/e8409f28347d5c688dfe8d45406be3512cefdf9e/src/MapKit/MKMapItem.cs#L112

Or maybe a method here with `[Wrap (...)]?`

Reply:
Or remove Internal and the _ I think this was left here by accident.